src/pages/AccountSettings/tabs/OktaAccess/OktaConfigForm/OktaConfigForm.test.tsx

```diff
@@ -312,7 +321,7 @@ describe('OktaConfigForm', () => {
         name: /Okta Login Enforced/,
       })
       await waitFor(() => {
-        expect(oktaLoginEnforceToggle).toHaveClass('bg-toggle-inactive')
+        expect(oktaLoginEnforceToggle).toHaveClass('bg-toggle-active')
```

Comment:
I had to update these tests, because beforehand they weren't using the data from the API query, as `suspense` wasn't enabled on the ` queryClient`, with it being controlled with which query hook you use now (i.e. `useSuspenseQuery`), these tests started failing, as in prod they would have suspended and used the values from the API.